pkg/sdk/testint/databases_integration_test.go

```diff
@@ -333,62 +811,88 @@ func TestInt_DatabasesShow(t *testing.T) {
+		})
 		require.NoError(t, err)
-		assert.Equal(t, 1, len(databases))
-		database := databases[0]
+
+		database, err := collections.FindOne(databases, func(database sdk.Database) bool { return database.Name == databaseTest.Name })
+		require.NoError(t, err)
+
 		assert.Equal(t, databaseTest.Name, database.Name)
 		assert.NotEmpty(t, database.CreatedOn)
 		assert.Empty(t, database.DroppedOn)
 		assert.Empty(t, database.Owner)
 	})
-	/*
-	   this test keeps failing, need to fix
-	   	t.Run("with history", func(t *testing.T) {
-	   		// need to drop a database to test if the "dropped_on" column is populated
-	   		databaseCleanup2()
-	   		showOptions := &ShowDatabasesOptions{
-	   			History: Bool(true),
-	   			Like: &Like{
-	   				Pattern: String(databaseTest2.Name),
-	   			},
-	   		}
-	   		databases, err := client.Databases.Show(ctx, showOptions)
-	   		require.NoError(t, err)
-	   		assert.Equal(t, 1, len(databases))
-	   		database := databases[0]
-	   		assert.Equal(t, databaseTest2.Name, database.Name)
-	   		assert.NotEmpty(t, database.DroppedOn)
-	   	})
-	*/
+
+	t.Run("with history", func(t *testing.T) {
+		databaseTest3, databaseCleanup3 := testClientHelper().Database.CreateDatabase(t)
+		databaseCleanup3()
+
+		databases, err := client.Databases.Show(ctx, &sdk.ShowDatabasesOptions{
+			History: sdk.Bool(true),
+			Like: &sdk.Like{
+				Pattern: sdk.String(databaseTest3.Name),
+			},
+		})
+		require.NoError(t, err)
+
+		droppedDatabase, err := collections.FindOne(databases, func(database sdk.Database) bool { return database.Name == databaseTest3.Name })
+		require.NoError(t, err)
+
+		assert.Equal(t, databaseTest3.Name, droppedDatabase.Name)
+		assert.NotEmpty(t, droppedDatabase.DroppedOn)
+	})
+
 	t.Run("with like starts with", func(t *testing.T) {
-		showOptions := &sdk.ShowDatabasesOptions{
+		databases, err := client.Databases.Show(ctx, &sdk.ShowDatabasesOptions{
 			StartsWith: sdk.String(databaseTest.Name),
 			LimitFrom: &sdk.LimitFrom{
 				Rows: sdk.Int(1),
 			},
-		}
-		databases, err := client.Databases.Show(ctx, showOptions)
+		})
 		require.NoError(t, err)
-		assert.Equal(t, 1, len(databases))
-		database := databases[0]
+
+		database, err := collections.FindOne(databases, func(database sdk.Database) bool { return database.Name == databaseTest.Name })
+		require.NoError(t, err)
+
 		assert.Equal(t, databaseTest.Name, database.Name)
 	})
 
 	t.Run("when searching a non-existent database", func(t *testing.T) {
-		showOptions := &sdk.ShowDatabasesOptions{
+		databases, err := client.Databases.Show(ctx, &sdk.ShowDatabasesOptions{
 			Like: &sdk.Like{
 				Pattern: sdk.String("non-existent"),
 			},
-		}
-		databases, err := client.Databases.Show(ctx, showOptions)
+		})
 		require.NoError(t, err)
+
 		assert.Equal(t, 0, len(databases))
 	})
 }
+
+func TestInt_DatabasesDescribe(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	databaseTest, databaseCleanup := testClientHelper().Database.CreateDatabase(t)
```

Comment:
why do we need a new database/schema here? Can't we just describe the one existing for tests?

Also, let's maybe check for different schemas here:
- the created one
- INFORMATION_SCHEMA
- PUBLIC

Reply:
Removed database creation and added suggested schema asserts